tests/rptest/utils/scale_parameters.py

```diff
@@ -19,6 +19,22 @@ class ScaleParameters:
     which are scale or performance tests, or which test some system limits need to be "scale aware"
     and can use ScaleParameters for that purpose."""
 
+    # How much memory is required per partition (from the partition memory pool
+    # specified by DEFAULT_PARTITIONS_MEMORY_ALLOCATION_PERCENT).
+    # Should be the same as cluster::DEFAULT_TOPIC_MEMORY_PER_PARTITION.
+    DEFAULT_MIB_PER_PARTITION = 200 / 1024  # 200 KiB
```

Comment:
did you mean `*` rather than `/`?

Reply:
No, I meant `/`. This gives ~0.2, i.e., 0.2 MiB which is 200 KiB which is the configured value for this setting.

Reply:
Oh, I see the name of the variable is `DEFAULT_MIB` not bytes.

Reply:
Yes exactly. Not sure if that's desirable, though in this case it's the same name used elsewhere for this property (this change is trying to consolidate those clones of this value into one place).

In the past this value was 4 MiB not 200 KiB which is probably why it has "MiB" as units.

Reply:
i think it's all good i just didn't read carefully enough